### PR TITLE
New buffers test

### PR DESCRIPTION
**Pull request by andpodob**
Update membrane rtp demo to work on membrane_core 0.8.0. This pipeline is an integration test that shows that support for PTS and DTS works in the `membrane_rtp_plugin`, `membrane_rtp_h264_plugin`, `membrane_h264_ffmpeg_plugin`, `membrane_rtp_opus_plugin`, `membrane_opus_plugin`.

### CODE REVIEW

File: rtp/lib/membrane_demo_rtp/receive_pipeline.ex [hunk 1]
```diff
@@ -37,21 +37,26 @@ defmodule Membrane.Demo.RTP.ReceivePipeline do
...
   @impl true
-  def handle_notification({:new_rtp_stream, _ssrc, encoding_name}, :rtp, _ctx, _state) do
+  def handle_notification(
+        {:new_rtp_stream, _ssrc, encoding_name, _extensions},
+        :rtp,
+        _ctx,
+        _state
+      ) do
     raise "Unsupported encoding: #{inspect(encoding_name)}"
   end
```
Comment (Qizot) [inline rtp/lib/membrane_demo_rtp/receive_pipeline.ex:62]: If I were you I would not ignore the unhandled notifications (the line below) :P

**Review by Qizot — APPROVED** [review]

File: rtp/lib/membrane_demo_rtp/receive_pipeline.ex (hunk 1 above)
Comment (andpodob) [reply rtp/lib/membrane_demo_rtp/receive_pipeline.ex:62]: There are some notifications from UDP sources, but I agree it's a better idea to match on them and log info that audio and video UDP sources were connected.

File: rtp/README.md [hunk 2]
```diff
@@ -33,10 +33,10 @@ The stream can also be sent with GStreamer:
 
 ```bash
 # For audio
-gst-launch-1.0 -v audiotestsrc ! lamemp3enc ! rtpmpapay pt=127 ! udpsink host=127.0.0.1 port=5000
+gst-launch-1.0 -v audiotestsrc wave=ticks ! audio/x-raw,format=S16LE,rate=48000,channels=2 ! opusenc ! rtpopuspay pt=120 ! udpsink host=127.0.0.1 port=5000
 
 # For video
-gst-launch-1.0 -v videotestsrc ! video/x-raw,format=I420 ! x264enc ! rtph264pay pt=96 ! udpsink host=127.0.0.1 port=5000
+gst-launch-1.0 -v videotestsrc pattern=ball ! video/x-raw,format=I420 ! x264enc ! rtph264pay pt=96 ! udpsink host=127.0.0.1 port=5000
```
Comment (mat-hek) [inline rtp/README.md:39]: We can make it a single command actually

```
gst-launch-1.0 -v audiotestsrc wave=ticks ! audio/x-raw,format=S16LE,rate=48000,channels=2 ! opusenc ! rtpopuspay pt=120 ! udpsink host=127.0.0.1 port=5000 \
    videotestsrc pattern=ball ! video/x-raw,format=I420 ! x264enc ! rtph264pay pt=96 ! udpsink host=127.0.0.1 port=5000
```

**Review by mat-hek — CHANGES_REQUESTED** [review]
I also get a warning
```
warning: use Mix.Config is deprecated. Use the Config module instead
  config/config.exs:1
```

**Review by Qizot — APPROVED** [review]

**Pull request merged**